### ISSUE TITLE
Convert-BcAppsToRuntimePackages Property Count not found

### ISSUE THREAD

**Issue by lktraser** [microsoft/navcontainerhelper#1675]
**Describe the issue**
I'm trying to use the new Convert-BcAppsToRuntimePackages but I'm running into a problem that I don't understand. I have the current bccontainerhelper 2..0.4 installed. When running the command I'm told the property count can't be found. My $apps variable has .Count and I also tried the Sort-AppFilesByDependencies on it as it's done in the function.

**Scripts used to create container and cause the issue**
```
$apps = @('C:\source\test.app')
Convert-BcAppsToRuntimePackages  `
    -containerName dmscorebuild  `
    -artifactUrl 'https://bcartifacts.azureedge.net/sandbox/17.3.20469.20825/de'  `
    -licenseFile 'C:\\license.flf'  `
    -apps $apps  `
    -destinationFolder 'C:\output\'
```

**Full output of scripts**
```
BcContainerHelper version 2.0.4
Convert-BcAppsToRuntimePackages : Die Eigenschaft "Count" wurde für dieses Objekt nicht gefunden. Vergewissern Sie sich, dass die Eigenschaft vorhanden ist.
In C:\test.ps1:2 Zeichen:1
+ Convert-BcAppsToRuntimePackages  `
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Convert-BcAppsToRuntimePackages], PropertyNotFoundException
    + FullyQualifiedErrorId : PropertyNotFoundStrict,Convert-BcAppsToRuntimePackages
...
```

Thanks for your support.

**Comment by freddydk** [comment]
Sounds like a bug

**Comment by freddydk** [comment]
The reason for this is that there is only one single app.
In my tests, I have been using a set of apps.
I have a fix for this.

**Comment by freddydk** [comment]
Fix shipped in 2.0.5-preview340
